### PR TITLE
Fixing GC

### PR DESCRIPTION
**Pull request by appaquet**

### CODE REVIEW

File: mry-core/src/main/scala/com/wajam/mry/storage/mysql/MysqlStorage.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ import java.util.concurrent.{TimeUnit, ScheduledThreadPoolExecutor}
  */
 class MysqlStorage(config: MysqlStorageConfiguration, garbageCollection: Boolean = true)
   extends Storage(config.name) with Logging with Value with Instrumented {
-  var model: Model = new Model
+  var model: Model = _
```
Comment (coderunner) [inline mry-core/src/main/scala/com/wajam/mry/storage/mysql/MysqlStorage.scala:19]: why this change?

Comment (appaquet) [inline mry-core/src/main/scala/com/wajam/mry/storage/mysql/MysqlStorage.scala:19]: Fixes a bug. The start function checks if model ==  null and throws if the model is null because it can't start without a model.

**Comment by coderunner** [comment]
+1
